homeassistant/components/voice_assistant/websocket_api.py

```diff
@@ -19,9 +30,11 @@ def async_register_websocket_api(hass: HomeAssistant) -> None:
 @websocket_api.websocket_command(
     {
         vol.Required("type"): "voice_assistant/run",
+        vol.Required("start_stage"): vol.In(set(PipelineStage)),
```

Comment:
Are you sure this is necessary? we've had enum validation before and didn't need this.

Reply:
This is what I get if I don't include the `set(...)`:

```
value must be one of [<PipelineStage.INTENT: 'intent'>, <PipelineStage.STT: 'stt'>, <PipelineStage.TTS: 'tts'>] for dictionary value @ data['start_stage']. Got 'tts'
```

Reply:
I don't know if this is a problem with the `StrEnum` backport or voluptuous

Reply:
I think you can just put the enum, not even use vol.In